src/styles/blocks/watch.scss

```
@@ -0,0 +1,38 @@
+@import "var.scss";
+
+.watch {
+  width: $width;
+  height: $height;
+  border-radius: 50%;
+  margin: 0 auto;
+  position: relative;
+  background: radial-gradient(circle,blue 5%, darkblue 88%, violet 88%);
+  &__second {
```

Apply a blank line between scss-rules and nested selectors
```suggestion
  background: radial-gradient(circle,blue 5%, darkblue 88%, violet 88%);
  
  &__second {
```